templates/cube/microcerts.html

```diff
@@ -0,0 +1,379 @@
+{% extends "cube/base_cube.html" %}
+
+{% block title %} Microcerts | CUBE {% endblock %}
+
+{% block meta_description %}This is the CUBE Microcerts page{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1IHmr2SIxcEKVNN2sA-z_32PV0fCXi38pmeVP7k7O1mM/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--square-suru has-cube is-dark is-slanted--bottom-left">
+  <div class="row u-equal-height u-vertically-center">
+    <div id="hero-content" class="col-6">
+      {% if certified_badge %}
+
+        <h1>CUBE 2020 complete</h1>
```

Comment:
2020?

Reply:
That's correct!

Reply:
Double checked with Lex, and this was his response: "Yes. We're tying the CUBE name to the LTS, not the year (although we're naming it by year), so "2020" matches "Ubuntu 20.0X""